Podfile.lock

```diff
@@ -46,4 +46,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 5a6fc3652ff27185dbe549b41dce70a850fa2ea3
 
-COCOAPODS: 1.7.0
+COCOAPODS: 1.6.1
```

Comment:
This probably shouldnt be in this PR

Reply:
I now have cocoapods 1.7.0 installed, & pushed revised lockfile which has caused it to drop out of this PR